standard/array.lua

```diff
@@ -623,4 +625,16 @@ function Array.unique(elements)
 	return uniqueElements
 end
 
+---@param inputString string?
+---@param options {makeLink: boolean?}?
+---@return string[]
+function Array.readCommaSeparatedList(inputString, options)
+	if Logic.isEmpty(inputString) then return {} end
+	options = options or {}
+	---@cast inputString -nil
+	local values = Array.map(mw.text.split(inputString, ','), String.trim)
+	if not options.makeLink then return values end
+	return Array.map(values, function(value) return Page.makeInternalLink({}, value) end)
```

Comment:
I think we should leave out the link making from this std function and have it quite generic and simple.